### ISSUE TITLE
Fix CSS to use correct selectors (was: Highlight own posts highlighting wrong posts on tinyboard/vichan)

### ISSUE THREAD

**Issue by ccd0** [ccd0/4chan-x#2169]
It doesn't show up when it should, and if you are the OP of a thread, it shows up for all replies.
The relevant CSS code is using 4chan-specific selectors.
We'll probably need a simple templating mechanism to insert selectors appropriate to the site.

**Comment by broski93** [comment]
Found two possibly related bugs on 8ch.net, so I'm posting here instead of opening a new issue.
1- Upon highlighting a thread in the filters menu, all the replies to that thread are highlighted as well (they get the solid bar on the left). That happens even if I specify op:only; and the posts did not contain the matched regex anyway. Seems like they inherit the highight css from the OP.

2- When posting, my post is recognized by the native extension (I get the (you)) and also in the 4chanX dropdown post menu (you) checkbox, but the post itself doesn't get the .yourPost attribute and therefore doesn't get marked by the dashed bar on the left. If I manually untick and tick again the (you) checkbox, everything works and I end up with the expected behavior.

Both tested on fresh FF Nightly profile, no addons besides Violentmonkey, and 4chanX beta script.
I've also experienced times when upon opening a watched thread some posts were marked as mine when they were not, but it happened at random and I was using my regular profile, so I haven't had the chance to try this one.
